core/eth/tx.go

```diff
@@ -686,7 +686,7 @@ func (t *Transactor) GetQuorumCount(ctx context.Context, blockNumber uint32) (ui
 	})
 }
 
-func (t *Transactor) GetQuorumSecurityParams(ctx context.Context, blockNumber uint32) ([]*core.SecurityParam, error) {
+func (t *Transactor) GetQuorumSecurityParams(ctx context.Context, blockNumber uint32) ([]core.SecurityParam, error) {
```

Comment:
Why is it changed from pointer to value?

Reply:
To make sure the result doesn't contain any nil. The number of security params will remain small as the values are managed by EigenDA governance